### ISSUE TITLE
Plot learner prediction broken

### ISSUE THREAD

**Issue by mllg** [mlr-org/mlr3viz#47]
This is supposed to work, right?

```r
library("mlr3")
library("mlr3viz")
gen = tgen("2dnormals")
gen$plot()

task = gen$generate(100)
plot_learner_prediction(lrn("classif.rpart"), task)
```

**Comment by mllg** [comment]
@jakob-r can you have a look?

**Comment by mllg** [comment]
Fixed via cc6e68112d790a8ee30529a62a40a1f0449ba851
